pulsar-transformations/src/main/resources/config-schema.yaml

```diff
@@ -44,6 +44,7 @@ components:
           compute: "#/components/schemas/Compute"
           compute-ai-embeddings: "#/components/schemas/ComputeAIEmbeddings"
           ai-chat-completions: "#/components/schemas/AIChatCompletions"
+          query: "#/components/schemas/Query"
```

Comment:
need a name more explicit about what type of query it is

Reply:
This is a "query", maybe we can name it "query-datasource" ?